src/test/java/com/hashedin/redmask/integration/redshift/RedMaskITTest.java

```diff
@@ -37,7 +36,7 @@
  * To run the integration test remove @Ignore annotation and
  * update Redshift credentials.
  */
-@Ignore
```

Comment:
Why `@Ignore` is removed?
This will break tests while running GitHub action, when PR is merged.

Reply:
added